Source/Core/JulianDate.js

```diff
@@ -634,7 +634,7 @@ define([
         }
         //>>includeEnd('debug');
 
-        var gDate = JulianDate.toGregorianDate(julianDate, gDate);
+        var gDate = JulianDate.toGregorianDate(julianDate);
```

Comment:
I think that this was actually supposed to be `gregorianDateScratch` instead of `gDate`.  So put that back as the parameter.